tests/fixtures/__init__.py

```diff
@@ -161,23 +161,24 @@ def run_task(
             for part in command_parts
         ]
 
-        if extra_args:
```

Comment:
These changes were required to support `&&` in the bash commands for the `run_task` abstraction.